parsl/executors/high_throughput/executor.py

```diff
@@ -599,6 +599,15 @@ def scale_in(self, blocks=None, block_ids=[]):
         blocks : int
              Number of blocks to terminate and scale_in by
 
+        force : Bool
+             Used along with blocks to indicate whether blocks should be terminated by force.
+             When force = True, we will kill blocks regardless of the blocks being busy
+             When force = False, we will kill only idle blocks
```

Comment:
When `force` is `False`, then `scale_in` will sometimes return without killing the requested number of blocks, and sometimes will kill no blocks at all, right?

Reply:
Yes. That's right, do you think the text needs updating to clarify this further ?

Reply:
yeah - it wasn't obvious to me

Reply:
@benclifford Can you check the docstring, please ?

Reply:
i think you should clarify that with force = false, this call will complete without killing the specified number of blocks and without throwing an error.